_includes/integrations/react-quick-start.md

```diff
@@ -96,22 +92,102 @@ This procedure requires:
+        To use an independent deployment of {{site.productname}}, add the property `tinymceScriptSrc` to specify the path to the {{site.productname}} script, such as:
+        ```jsx
+        <Editor tinymceScriptSrc="/path/to/tinymce.min.js" />
+        ```
+
+        To do this with create-react-app put the tinymce distribution in `./public`
+        and reference the path to the `public` folder using the environment 
+        variable `process.env.PUBLIC_URL`, such as:
+         ```jsx
+         <Editor tinymceScriptSrc={process.env.PUBLIC_URL + '/tinymce/tinymce.min.js'}>
+         ```
+
+      * **Deploy {{site.productname}} independent of the React application with a script tag**
+
+        An alternative method is to add a script to either the `<head>` or the
+        end of the `<body>` of the HTML file, such as:
         ```html
         <script src="/path/to/tinymce.min.js"></script>
         ```
 
-        To use an independent deployment of {{site.productname}} with the create a React application, add the script to `/path/to/tinymce-react-demo/public/index.html`.
+        To do this with create-react-app, add the script tag to `./public/index.html`.
+        
+        Normally the tinymce distribution would be put in the `public` folder
+        and referenced using the URL `%PUBLIC_URL%/tinymce/tinymce.min.js`, such as:
+        ```html
+        <script src="%PUBLIC_URL%/tinymce/tinymce.min.js"></script>
+        ```
 
-        For information on self-hosting {{site.productname}}, see: [Installing {{ site.productname }}]({{site.baseurl}}/general-configuration-guide/advanced-install/).
+        For information on self-hosting {{site.productname}}, see: [Installing {{site.productname}}]({{site.baseurl}}/general-configuration-guide/advanced-install/).
 
       * **Bundling {{site.productname}} with the React application using a module loader**
 
+          Bundling with a module loader can be complex and error prone so it is
+          not recommended.
+
           To bundle {{site.productname}} using a module loader (such as Webpack and Browserify), see: [Usage with module loaders]({{site.baseurl}}/advanced/usage-with-module-loaders/).
 
-7. Test the application using the Node.js development server.
+          Example of bundling:
+          ```jsx
+          import { Editor } from '@tinymce/tinymce-react';
+
+          // TinyMCE so the global var exists
+          // eslint-disable-next-line no-unused-vars
+          import tinymce from 'tinymce/tinymce';
+
+          // Theme
+          import 'tinymce/themes/silver';
+          // Toolbar icons
+          import 'tinymce/icons/default';
+          // Editor styles
+          import 'tinymce/skins/ui/oxide/skin.min.css';
+
+          // importing the plugin js. Note that most of the premium plugins can't be imported this way because they have extra resources that have to be loaded asynchronously at runtime
```

Comment:
Is this really true? As far as I know all of them work with the exception of some CSS files (eg comments or advcode). I'd suggest removing or rewording this (eg some instead of most) as its not really accurate and a "detractor" statement which normally is frowned upon.

Reply:
Ok, will remove.